src/util/redisConnector.js

```diff
@@ -3,10 +3,10 @@ const { RedisError } = require('../v0/util/errorTypes');
 const log = require('../logger');
 const stats = require('./stats');
 
-const timeoutPromise = new Promise((resolve, reject) => {
```

Comment:
@anantjain45823 check out this code fix & test out in local on the connection scenarios